src/ppl/test/annealing2.ok

```diff
@@ -10,5 +10,5 @@
 Found 0 macro blocks.
 Using 1u default distance from corners.
 Using 2 tracks default min distance between IO pins.
-[INFO PPL-0012] I/O nets HPWL: 1962.48 um.
+[INFO PPL-0012] I/O nets HPWL: 1962.64 um.
```

Comment:
Are you expecting a change in this test which doesn't use groups?

Reply:
This test has two pin groups.